packages/core/src/pipeline/config/graph/PipelineGraph.tsx

```diff
@@ -359,24 +368,40 @@ export class PipelineGraph extends React.Component<IPipelineGraphProps, IPipelin
     newState.graphHeight = Math.max(sum(newState.rowHeights) + this.graphVerticalPadding, this.minExecutionGraphHeight);
   }
 
-  private updateGraph(props: IPipelineGraphProps, statesOnly = false): void {
+  private updateGraph(props: IPipelineGraphProps, expand = false, statesOnly = false): void {
     let newState: IPipelineGraphState;
-    if (!statesOnly) {
+
```

Comment:
Indentation + extra new line - might be worth running lint and/or spotless:apply.